engine/src/test/java/org/camunda/bpm/engine/test/bpmn/deployment/BpmnDeploymentTest.java

```diff
@@ -177,7 +185,7 @@ public void shouldLogWarningForDuplicateFilteringWithoutPreviousDeploymentName()
         .addDeploymentResources(deployment.getId());
 
     // when
-    testRule.deploy(deploymentBuilder);
+    DeploymentWithDefinitions deployment2 = testRule.deploy(deploymentBuilder);
```

Comment:
This will not be needed anymore, I guess?
```suggestion
    testRule.deploy(deploymentBuilder);
```